pkg/umorse/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config USEPKG_UMORSE
+    bool "umorse"
+    select USEMODULE_POSIX_SLEEP
```

Comment:
```suggestion
    select MODULE_POSIX_SLEEP
```

Reply:
We should check if it is safe to keep the `select` or this should be `depends on`. The posix modules are not modelled AFAIK, so those should be added.